src/JSValue.Parsing.swift

```diff
@@ -465,11 +464,18 @@ extension JSValue {
                                 }
 
                                 let codepoint = (value1! << 12) | (value2! << 8) | (value3! << 4) | value4!;
-                                let character = String(UnicodeScalar(codepoint))
-                                let data = character.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: false)!
-                                let ptr = UnsafePointer<UInt8>(data.bytes)
-                                let escapeBytes = UnsafeBufferPointer<UInt8>(start: ptr, count: data.length)
-                                bytes.appendContentsOf(escapeBytes)
+                                if let scalar = UnicodeScalar(codepoint) {
```

Comment:
This is really the only manual change I had to do. I think it works the same but please pay extra attention when reviewing this.